load.php

```diff
@@ -236,6 +237,11 @@ function perflab_can_load_module( $module ) {
 		return true;
 	}
 
+	// Do not load the module if it can be loaded by a separate plugin.
+	if ( perflab_is_module_standalone( $module ) ) {
+		return false;
+	}
```

Comment:
Based on my comment https://github.com/WordPress/performance/pull/670#discussion_r1139423419, can you please remove this from here to not intertwine it with the `perflab_can_load_module()` function? Instead, the new function should be called wherever `perflab_can_load_module()` is called (i.e. in `perflab_is_valid_module()`).